n3fit/src/evolven3fit_new/eko_utils.py

```diff
@@ -34,16 +36,37 @@ def construct_eko_cards(
     theory = Loader().check_theoryID(theoryID).get_description()
     theory.pop("FNS")
     theory.update(theory_card_dict)
-    theory_card = gen_theory.gen_theory_card(theory["PTO"], theory["Q0"], update=theory)
+    if "nfref" not in theory:
+        theory["nfref"] = 5
+    if "nf0" not in theory:
+        theory["nf0"] = 4
+    legacy_class = runcards.Legacy(theory, {})
+    theory_card = legacy_class.new_theory
```

Comment:
What does this legacy class do?

Reply:
This is unfortunately the simplest way I found to read the nnpdf-like theory card (and mimic what is done in pineko) ... 
@AleCandido might have better suggestions

Reply:
This is the same class that we are using in `pineko`. It is able basically to read a NNPDF theory card and convert it to an eko card.

Reply:
But it is something to improve

Reply:
Ah, ok. Please add a comment on top "Parse the NNPDF theory information into the eko theory" or something.